src/libs/isReportMessageAttachment.ts

```diff
@@ -1,13 +1,18 @@
 import CONST from '../CONST';
 
+type IsReportMessageAttachmentArguments = {
```

Comment:
```suggestion
type IsReportMessageAttachmentParams = {
```

Could we rename it? We have been using `Params` suffix.

Reply:
Fixed @fabioh8010